### PR TITLE
Document *-jumps-to-next-unread settings

### PR DESCRIPTION
**Pull request by tsipinakis**
> Add missing documentation for openbrowser-jumps-to-next-unread and
> toggleitemread-jumps-to-next-unread config commands.

Additional note: There is also markfeedread-jumps-to-next-unread but I haven't been able to get it to work so I didn't add any documentation for that.

### CODE REVIEW

File: doc/configcommands.dsv [hunk 1]
```diff
@@ -73,6 +73,7 @@ oldreader-min-items|<number>|20|This variable sets the number of articles that a
 oldreader-password|<password>|""|This variable sets your The Old Reader password for The Old Reader support. Double quotes should be escaped, i.e. you should write +{backslash}"+ instead of +"+.|oldreader-password "here_goesAquote:{backslash}""
 oldreader-passwordfile|<path-to-file|""|A more secure alternative to the above, by storing your password elsewhere in your system.|oldreader-passwordfile "path-to-file"
 oldreader-show-special-feeds|[yes/no]|yes|If this is set, then "special feeds" like "People you follow" (articles shared by people you follow), "Starred items" (your starred articles) and "Shared items" (your shared articles) appear in your subscription list.|oldreader-show-special-feeds "no"
+openbrowser-and-mark-jumps-to-next-unread|[yes/no]|no|If set to yes, jump to the next unread item when an item is opened to the browser and marked as read(Default: O).|openbrowser-and-mark-jumps-to-next-unread yes
```
Comment (Minoru) [inline doc/configcommands.dsv]: 1. An item is opened *in* the browser, not *to* it.
2. We already have the default value in the third column, so no need to put "(Default: 0)" at the end of the description.

File: doc/configcommands.dsv [hunk 2]
```diff
@@ -98,6 +99,7 @@ show-read-feeds|[yes/no]|yes|If yes, then all feeds, including those without unr
 suppress-first-reload|[yes/no]|no|If yes, then the first automatic reload will be suppressed if auto-reload is set to yes.|suppress-first-reload yes
 swap-title-and-hints|[yes/no]|no|If yes, then the title at the top of screen and keymap hints at the bottom of screen will be swapped.|swap-title-and-hints yes
 text-width|<number>|0|If set to a number greater than 0, then all HTML will be rendered to this maximum line length. If set to 0, the terminal width will be used. Does not apply when using external renderer or viewing the source. Also note that "Link" header and "Links" section won't be affected by it—they contain URLs which are better not wrapped.|text-width 72
+toggleitemread-jumps-to-next-unread|[yes/no]|no|If set to yes, jump to the next unread item when an items read status is toggled in the articlelist(Default: N).|toggleitemread-jumps-to-next-unread yes
```
Comment (Minoru) [inline doc/configcommands.dsv]: 1. "an *item's s* status", not "an *items* status".
2. Again, no need to put default value in the description.

**Comment by tsipinakis** [comment]
Fixed the wording, still not sure how to mark reviews as done with the new github system.

I actually stumbled upon one of them by chance and made me wonder the same thing, so I used a simple unix one-liner to check for any more.

Results:
```
$ diff <(cat doc/configcommands.dsv doc/podbeuter-cmds.dsv | cut -d"|" -f1 | sort | uniq) <(cut -d" " -f1 conf | sort | uniq)
2d1
< always-download
14d12
< color
18d15
< define-filter
34a32
> feedhq-url
43d40
< highlight-article
47d43
< ignore-article
49d44
< include
53a49
> markfeedread-jumps-to-next-unread
59a56
> newsblur-url
92d88
< reset-unread-on-update
113d108
< unbind-key
```
Those marked as removed from the diff don't actually appear in dumpconfig.

From this we still have undocumented: `feedhq-url`, `markfeedread-jumps-to-next-unread` amd, `newsblur-url`

**Comment by Minoru** [comment]
> still not sure how to mark reviews as done with the new github system

The author (you) don't. You simply submit the changes and then wait for reviewer(s) (me) to press the button to approve the changes.

> Those marked as removed from the diff don't actually appear in dumpconfig.

I consider this a bug! Will file an issue in a moment.

> From this we still have undocumented: `feedhq-url`, `markfeedread-jumps-to-next-unread` and, `newsblur-url`.

`feedhq-url` is in configcontainer only for convenience, I guess—since it can't be self-hosted, there's no point in providing users with a way to change the service's URL.

Would you like to document the other two (either as part of this PR or a separate one)? If not, I'll merge this PR as is and file issues for the other two settings.

**Comment by tsipinakis** [comment]
I added the `newsblur-url` documentation.
As I said I wasn't able to figure out how `markfeedread-jumps-to-next-unread` works and don't the time to debug it right now.

**Comment by Minoru** [comment]
Sorry, my attention span must be really short today—I already forgot that you mentioned that other setting in the PR ._. I'll file an issue about it, then, to pick it up later.

Thanks for your work!

**Pull request merged**